lib/puppet/provider/package/pkgin.rb

```diff
@@ -12,8 +12,8 @@
   def self.parse_pkgin_line(package)
 
     # e.g.
-    #   vim-7.2.446 =        Vim editor (vi clone) without GUI
-    match, name, version, status = *package.match(/(\S+)-(\S+)(?: (=|>|<))?\s+.+$/)
+    #   vim-7.2.446;Vim editor (vi clone) without GUI
+    match, name, version, status = *package.match(/(\S+)-(\S+?);(?:(=|>|<))?.+$/)
```

Comment:
Should we have the regex then support either format so that we don't break users running an older pkgin?  Something like: `/(\S+)-([^\s;=><]+)(?: ?[;=><])?\s*.+$/`?

Also, I don't see a capture group for `status`. Is that always nil?